pkg/ccl/crosscluster/replicationtestutils/testutils.go

```diff
@@ -284,12 +284,10 @@ func (c *TenantStreamingClusters) Cutover(
 	require.LessOrEqual(c.T, protectedTimestamp.GoTime(), cutoverOutput.GoTime())
 
 	// PTS should be less than or equal to retained time as a result of heartbeats.
-	var retainedTime time.Time
-	c.DestSysSQL.QueryRow(c.T,
-		`SELECT retained_time FROM [SHOW TENANT $1 WITH REPLICATION STATUS]`,
-		c.Args.DestTenantName).Scan(&retainedTime)
-
-	require.LessOrEqual(c.T, protectedTimestamp.GoTime(), retainedTime)
+	retainedTime := replicationutils.TestingGetRetainedTimeFromReplicationJob(c.T, ctx, c.SrcSysSQL, c.SrcSysServer, producerJobID)
```

Comment:
this scan was flaking because we couldn't cast a nil value to a timestamp right? Instead of building a brand new helper function, could we just handle the error after the query? i.e.:

```
if err := db.QueryRow(blah).Scan(&retainedTime); err != nil {
  if err is nil, skip the pts to retained time check
}
```

Reply:
I wonder if it's better to query pts on the destination cluster directly since `Scan()` kills the test on error and doesn't return anything?

Reply:
I don't know if it'll work at this call site, but there is also `pq.NullTime` which you could scan into.

Reply:
Awesome I think that's what we need, thanks for the suggestion!